openfl/component/aggregator/aggregator.py

```diff
@@ -537,7 +550,8 @@ def send_local_task_results(self, collaborator_name, round_number, task_name,
 
         self._end_of_task_check(task_name)
 
-    def _process_named_tensor(self, named_tensor, collaborator_name):
+    def _process_named_tensor(self, named_tensor: Any,
+                              collaborator_name: str) -> Union[Tuple[Tuple, ndarray], NoReturn]:
```

Comment:
Strange construction here) `NoReturn` means the function never return a value.
About first returned argument, can we just annotate it by the following:
```python3
from openfl.utilities import TensorKey
```

```suggestion
                              collaborator_name: str) -> Tuple[TensorKey, ndarray]:
```